src/database/book.sql

```diff
@@ -0,0 +1,6 @@
+
+CREATE TABLE BOOK (
+ID BIGINT NOT NULL  AUTO_INCREMENT, -- BOOK COPY ID
```

Comment:
this is not book copy id. this is book id.

Reply:
done